toptal/gitignore.io
# [SAT-184] Allow creating without selecting first

[SAT-184](https://toptal-core.atlassian.net/browse/SAT-184)
## Descriptiont

Using the selectOnClose option, select2 will automatically select the
entry if the entered string exists on the list

## How to test

- docker-compose -f ./docker-compose-dev.yml up --build
- go to http://localhost:8080
- Enter a valid entry on the main Input but don't select it. eg. `python`
- Click on the "Create" button. Python should've been automatically selected from the list and the create button works.
